core/controllers/tasks.py

```diff
@@ -76,6 +77,21 @@ def post(self):
 class SuggestionEmailHandler(base.BaseHandler):
     """Handler task of sending email of suggestion."""
 
+    URL_PATH_ARGS_SCHEMAS = {}
+    HANDLER_ARGS_SCHEMAS = {
+        'POST': {
+            'exploration_id': {
+                'schema': editor.SCHEMA_FOR_EXPLORATION_ID
+            },
+            'contributor_user_id': {
+                'schema': {
+                    'type': 'basestring'
+                },
+                'default_value': None
```

Comment:
Are there more validations possible?

Reply:
Simply added "'default_value' : None" since it was done on line 122 in the same file. Should i remove it? Or throw exception if it is None?

Reply:
Or did you mean something else with validations?

Reply:
Are there more possible validations for this field, like can we limit its max length or maybe it has some structure that it needs to follow.

Reply:
I see what you mean now, thank you! I checked if other schemas made for 'contributor_user_id' had any validators and they didn't. I also checked the same thing for other types of 'basestring'-ids but but did not find any suitable validators (only found validators for regex-based schemas). Should I add a validator anyway? Perhaps with 'id': 'has_length_at_most' and with a suitable 'max_size'?

Reply:
Can you look for other `user_id` fields?

Reply:
@vojtechjelinek Changed 'contributor_user_id' to 'thread_id'.